File: UNIT 4/Python/array_string.py
nama = input('Masukan nama anda : ')
jk = 0
jb = 0

for ch in nama:
    if (ord(ch) >= 97 and ord(ch) <= 122):
        jk += 1
    elif (ord(ch) >= 65 and ord(ch) <= 90):
        jb += 1

print("Nama :", nama)
print("Jumlah huruf kecil :", jk)
print("Jumlah huruf besar :", jb)
print("Jumlah huruf dan spasi :", len(nama))

File: UNIT 4/C/array_string.c
#include <stdio.h>


int main() {
    char nama[17];
    int ch, maks = 17, nch = 0;
    int jk = 0, jb = 0;
    
    maks = maks - 1; // Sisakan 1 byte untuk null terminator
    
    printf("Masukan nama anda: ");
    
    while ((ch = getchar()) != EOF) {
        if (ch == '\n')
            break;
        
        if (nch < maks) {
            nama[nch] = ch;
            if (nama[nch] >= 'a' && nama[nch] <= 'z')
                jk++;
            else if (nama[nch] >= 'A' && nama[nch] <= 'Z')
                jb++;
        }
        nch = nch + 1;
    }
    
    nama[nch] = '\0'; // Menambahkan null terminator
    
    printf("Nama : %s \n", nama);
    printf("Jumlah huruf kecil : %d \n", jk);
    printf("Jumlah huruf besar : %d \n", jb);
    printf("Jumlah huruf dan spasi : %d \n", nch);
    
   
    return 0;
}